src/progress_bars/ProgressBars.test.tsx

```diff
@@ -0,0 +1,166 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import {ProgressBar, ProgressBarProps, ProgressBarType, ProgressBarStatus, ProgressBarAnimation} from ".";
+import {shallow, mount, ShallowWrapper} from "enzyme";
+import * as React from "react";
+
+let wrapper: ShallowWrapper<ProgressBarProps>;
+
+describe("<ProgressBar /> rendering", () => {
+    beforeAll(() => {
+        wrapper = shallow<ProgressBar>(<ProgressBar value={10} max={100} style={{width: "50%"}} />);
+    });
+
+    it("renders correctly", () => {
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("<ProgressBar /> rendering with label", () => {
+    beforeAll(() => {
+        wrapper = shallow<ProgressBar>(<ProgressBar value={10} max={100} labeled={true} style={{width: "50%"}} />);
+    });
+
+    it("renders correctly", () => {
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("<ProgressBar /> rendering with static progress", () => {
+    beforeAll(() => {
+        wrapper = shallow<ProgressBar>(
+            <ProgressBar value={10} max={100} type={ProgressBarType.STATIC} style={{width: "50%"}} />,
+        );
+    });
+
+    it("renders correctly", () => {
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("<ProgressBar /> rendering with Labeled, Static Progress Bar", () => {
+    beforeAll(() => {
+        wrapper = shallow<ProgressBar>(
+            <ProgressBar value={50} max={100} type={ProgressBarType.STATIC} labeled={true} style={{width: "50%"}} />,
+        );
+    });
+
+    it("renders correctly", () => {
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("<ProgressBar /> rendering with Indeterminate (Looping)", () => {
+    beforeAll(() => {
+        wrapper = shallow<ProgressBar>(
+            <ProgressBar value={100} max={100} className={ProgressBarAnimation.LOOP} style={{width: "50%"}} />,
+        );
+    });
+
+    it("renders correctly", () => {
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("<ProgressBar /> rendering with animation for flash then fade", () => {
+    beforeAll(() => {
+        wrapper = shallow<ProgressBar>(
+            <ProgressBar
+                value={100}
+                max={100}
+                className={`${ProgressBarAnimation.FADE_OUT}  ${ProgressBarAnimation.FLASH}`}
+                style={{width: "50%"}}
+            />,
+        );
+    });
+
+    it("renders correctly", () => {
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("<ProgressBar /> rendering with animation for red flash and no fade", () => {
+    beforeAll(() => {
+        wrapper = shallow<ProgressBar>(
+            <ProgressBar value={100} max={100} className={ProgressBarAnimation.FLASH_DANGER} style={{width: "50%"}} />,
```

Comment:
you can also test with 'max' value other than 100. Do not confuse this with percentage.

Reply:
Addressed the comment